provider/terraform/async.rb

```diff
@@ -31,12 +36,18 @@ class PollAsync < Api::Async
       # result of the final Read()
       attr_reader :suppress_error
 
+      # Number of times the desired state has to occur continuously
+      # during polling before returning a success
+      attr_reader :target_occurrences
+
       def validate
         super
 
-        check :check_response_func, type: String, required: true
+        check :check_response_func_existence, type: String, required: true
+        check :check_response_func_absence, type: String, default: 'PollCheckForAbsence'
```

Comment:
Can we make this `required` similar to the `existence` field? No particular reason other than keeping it consistent